<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>上传状态</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dataVolt">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>120</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>120</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ADC</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dataAccX">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>150</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>150</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>AccX</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dataGyroX">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>150</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>150</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>GyroX</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dataAccY">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>180</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>180</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>AccY</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dataAccZ">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>210</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>210</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>AccZ</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>180</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>GyroY</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dataGyroY">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>180</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>210</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>GyroZ</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dataGyroZ">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>210</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>240</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pitch</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dataPitch">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>240</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>270</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Roll</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dataRoll">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>270</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>300</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Yaw</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dataYaw">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>300</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="stateLED1">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>30</y>
     <width>87</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>LED1</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="stateLED2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>50</y>
     <width>87</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>LED2</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="stateLED3">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>70</y>
     <width>87</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>LED3</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="stateLED4">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>90</y>
     <width>87</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>LED4</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="stateKEY1">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>30</y>
     <width>87</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>KEY1</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="stateKEY2">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>50</y>
     <width>87</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>KEY2</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="stateKEY3">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>70</y>
     <width>87</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>KEY3</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="stateKEY4">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>90</y>
     <width>87</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>KEY4</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>接收窗口</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textRecv">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>32</y>
     <width>441</width>
     <height>191</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>227</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>发送窗口</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="textSend">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>248</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>50</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>端口号：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>290</y>
     <width>70</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>70</width>
     <height>16777215</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>COM1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM4</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>290</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>16777215</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>1200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4800</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>38400</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>115200</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>290</y>
     <width>50</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>波特率：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>290</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>bps</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSend">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>240</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOpen">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>280</y>
     <width>115</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>115</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>打开串口</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClear">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>240</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dataVolt_2">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>120</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>120</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>电压</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
